.circleci/config.yml

```diff
@@ -359,7 +359,7 @@ workflows:
       - build_and_push:
           requires:
           - lint_checks
-      - auto_deploy_uat:
+      - uat_master_deploy:
```

Comment:
This should be left as auto_deploy_uat because it's deploying the open_pr's branch to UAT e.g. ap-1xx-uat